client/README.md

```diff
@@ -0,0 +1,43 @@
+# Flavor Finder Interface
+
+Find your place to eat
+
+## Get Started
+
+Clone the project
+
+```bash
+git clone https://github.com/aytmamatov/flavor_finder.git
+```
+
+Go to the client folder
```

Comment:
Specify that there is frontend setup, i guess there is no need to create `README` file in the client folder, can we update the main README in the root?